File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/HeaderItemDecoration.java
package com.xwray.groupie.example.viewbinding;

import androidx.annotation.ColorInt;
import androidx.annotation.LayoutRes;

public class HeaderItemDecoration extends com.xwray.groupie.example.core.decoration.HeaderItemDecoration {
    public HeaderItemDecoration(@ColorInt int background, int sidePaddingPixels) {
        super(background, sidePaddingPixels, R.layout.item_header);
    }
}

File: example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/HeaderItemDecoration.kt
package com.xwray.groupie.example.viewbinding

import androidx.annotation.ColorInt
import com.xwray.groupie.example.core.decoration.HeaderItemDecoration

class HeaderItemDecoration(
    @ColorInt background: Int,
    sidePaddingPixels: Int
) : HeaderItemDecoration(background, sidePaddingPixels, R.layout.item_header)
